src/Fieldtypes/BaseFieldtype.php

```diff
@@ -259,4 +258,13 @@ protected function getCreatables()
             ],
         ];
     }
+
+    protected function makeTitle($record, $resource, string $default): string
+    {
+        if (! $titleFormat = $this->config('title_format')) {
+            return $record->$default;
```

Comment:
Instead of `$default` being passed into this method, could we instead simply use the first listable column?